src/modules/mahjong/MahjongPlayer/MahjongPlayer.ts

```diff
@@ -179,12 +190,15 @@ class MahjongPlayer extends UserEntity {
   public renderInteractions(
```

Comment:
this method seems a bit too big, can consider splitting some of the logic into smaller functions

Reply:
Will refactor